client/src/components/creative-colab.css

```diff
@@ -0,0 +1,10 @@
+.editor, .story-container { 
+  width: 500px;
+  height: 300px;
+  max-width: 500px;
+  max-height: 300px;
+}
+
+.story-container {
+  overflow-y: scroll;

```

Comment:
Great! This is exactly what we need!

Reply:
We'd also need to add `overflow-wrap: break-word;` to prevent horizontal scrolling